.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,16 @@
+Thank you for reporting and issue to google-cloud-python!
+
+If you are reporting an issue or requesting a feature, please search the
+existing open and closed issues to see if there is already work being done.
+
+If you can provide us with the following information it will help us identify
+the cause of your issue more quickly.
+
+1. google-cloud-python version
+2. Python version and virtual environment information
+3. Stacktrace if available
+4. Steps to reproduce
+5. Code example
+
+Using Github flavored markdown can help make your request clearer.
+See: https://guides.github.com/features/mastering-markdown/
```

Comment:
Markdown in this doesn't get parsed. It shows up as raw text in the issue textarea box.
See: https://github.com/GoogleCloudPlatform/google-cloud-node/issues/new